lib/events/auditlog.go

```diff
@@ -31,6 +31,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/google/uuid"
```

Comment:
I noticed we also used `github.com/pborman/uuid` to generate UUIDs (https://github.com/gravitational/teleport/blob/master/lib/auth/grpcserver.go#L235). Is there a reason we use different generators or can we just get rid of one and use the other across the project?

Reply:
I'm putting up #9793 to switch to google's package across the codebase.